src/Admin/Models/OrganizationEditModel.cs

```diff
@@ -78,6 +79,8 @@ public class OrganizationEditModel : OrganizationViewModel
         public bool UsePolicies { get; set; }
         [Display(Name = "SSO")]
         public bool UseSso { get; set; }
+        [Display(Name = "Customer Managed Encryption")]
```

Comment:
We use `Key Connector` in the web vault interface, should we use the same wording here for consistency?

Reply:
Hmm, valid point. My thinking was that the feature is called "Customer Managed Encryption" which might make it more obvious to use that name vs the internal. I'll check with Product.